### DISCUSSION TITLE
best way to find app details when traditional methods don't work

### DISCUSSION THREAD

**Discussion by grrrrr** [Arcticons-Team/Arcticons#1153] [Q&A]
The icon for [Nova Launcher Prime](https://play.google.com/store/apps/details?id=com.teslacoilsw.launcher.prime) is not working with Arcticons

I am trying to find a way to get the correct info to update `appfilter.txt` to do a pull request but I am having issues getting the correct content.   `Arcticons`, `Turtl` and `Icon Request for Numix` do not see Nova Prime as an installed App all I see is Nova Launcher 

Are there any other suggested ways to extract the needed info?

**Comment by Donnnno** [comment]
I remembered finding the correct line of code somewhere, but never applied it. I will search again. I do believe this is a very rare case with nova launcher.
